massgov/mayflower-patternlab
# Info Details sticky TOC fixes

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description
In testing long form content types (binder and info details), it was found that the internal binder bottom navigation (previous and next) component expanded the full width of the page.  This addresses that.  It was also found that the sticky TOC was getting items from the sidebar, not just the main content.  It was also found that the section h2 styles were covering up links just above, making them unable to be clicked.  This update prevents that.

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8406)

## Steps to Test


1. Checkout the branch locally and run Mayflower locally.
2. Confirm that inner binder navigation page navigation is confined to the width of the content + sidebar instead of going full width (http://localhost:3000/?p=pages-binder-page-internal).
3. Confirm that the TOC still works properly (http://localhost:3000/?p=pages-information-details-table-of-contents).

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
